src/extlibs/er_force_sim/src/amun/simulator/simball.cpp

```diff
@@ -69,31 +69,72 @@ SimBall::~SimBall()
     delete m_motionState;
 }
 
-void SimBall::begin()
+void SimBall::begin(bool robotCollision)
 {
     // custom implementation of rolling friction
-    const btVector3 p = m_body->getWorldTransform().getOrigin();
+    const btVector3 p        = m_body->getWorldTransform().getOrigin();
+    const btVector3 velocity = m_body->getLinearVelocity();
     if (p.z() < BALL_RADIUS * 1.1 * SIMULATOR_SCALE)
     {  // ball is on the ground
-        const btVector3 velocity = m_body->getLinearVelocity();
-        if (velocity.length() < 0.01 * SIMULATOR_SCALE)
+        BallState nextState = currentBallState;
+        bool isStationary   = velocity.length() < STATIONARY_BALL_SPEED * SIMULATOR_SCALE;
```

Comment:
usually our variables are snake_case

Reply:
is it worth breaking this rule to be consistent with the rest of the er force code